wagtailvideos/models.py

```diff
@@ -50,7 +54,7 @@ class AbstractVideo(CollectionMember, TagSearchable):
         verbose_name=_('file'), upload_to=get_upload_to)
     thumbnail = models.ImageField(upload_to=get_upload_to, null=True, blank=True)
     created_at = models.DateTimeField(verbose_name=_('created at'), auto_now_add=True, db_index=True)
-    duration = models.CharField(max_length=255, blank=True)
+    duration = models.DurationField(blank=True, null=True)
```

Comment:
I assume this field is printed in some templates somewhere. The `str` representation of a `timedelta` is rather ugly, so this has probably broken any templates where it is printed out. A `formatted_duration` `@property` on the `AbstractVideo` model that does the job of the string formatting code you removed would be useful, as formatting a `timedelta` in a template sounds not fun.